packages/shared/src/hooks/streaks/useReadingStreak.ts

```diff
@@ -2,26 +2,29 @@ import { useQuery } from '@tanstack/react-query';
 import { generateQueryKey, RequestKey, StaleTime } from '../../lib/query';
 import { getReadingStreak, UserStreak } from '../../graphql/users';
 import { useAuthContext } from '../../contexts/AuthContext';
-import { useStreakExperiment } from './useStreakExperiment';
+import { useActions } from '../useActions';
+import { ActionType } from '../../graphql/actions';
 
 interface UserReadingStreak {
   streak: UserStreak;
   isLoading: boolean;
-  isEnabled: boolean;
+  shouldShowPopup: boolean;
 }
 
 export const useReadingStreak = (): UserReadingStreak => {
   const { user } = useAuthContext();
-  const { shouldShowStreak } = useStreakExperiment();
   const { data: streak, isLoading } = useQuery(
     generateQueryKey(RequestKey.UserStreak, user),
     getReadingStreak,
-    { enabled: shouldShowStreak, staleTime: StaleTime.Default },
+    { enabled: true, staleTime: StaleTime.Default },
   );
+  const { checkHasCompleted } = useActions();
 
   return {
     streak,
     isLoading,
-    isEnabled: shouldShowStreak,
+    shouldShowPopup:
+      user?.createdAt < '2024-03-14T00:00:00.000Z' &&
```

Comment:
What are we doing here?

Reply:
Tnx for explanation!

Reply:
Are there even people still that where from before this date and haven't seen this yet? 🔱  (In which case we can also remove the modal?)

Reply:
Hmm will need to check, would be nice to remove

Reply:
I checked an not all existing users have seen this